src/docs/markdown/caddyfile/directives/bind.md

```diff
@@ -16,10 +16,13 @@ Note that binding sites inconsistently may result in unintended consequences. Fo
 ## Syntax
 
 ```caddy-d
-bind <hosts...>
+bind <hosts...> {
+	protocols <protocol> ...
```

Comment:
Is it moreseo like this?

```suggestion
	protocols <protocols...>
```

`<protocol> ...` implies a single protocol is given, which can itself have additional options, but from my reading of the examples, it's just a list of protocols.

Reply:
with the `...` after, I thought this was the notation for at least one protocol, then possibly more. there are other options that use both, but I thought the distinction was >=1 vs >=0.

Reply:
We use `<foos...>` to to mean "one or more". See https://caddyserver.com/docs/caddyfile/directives#syntax. If it was optional, it would be `[<foos...>]`.